requirements.txt

```diff
@@ -2,7 +2,7 @@ Django>=2.2,<3.0
 django-geostore>=0.3,<0.4
 requests>=2.19,<2.20
 bonobo>=0.6.0,<0.6.9
-elasticsearch>=0.6.0,<7.0.0
+elasticsearch>=0.6.0,<8.0.0
```

Comment:
Why `0.6.0` here ? You should force minimum version to avoid keeping currently installed package.